src/couch/src/couch_httpd_auth.erl

```diff
@@ -253,6 +246,68 @@ jwt_authentication_handler(Req) ->
             Req
     end.
 
+get_roles_claim(Claims) ->
+    Roles_Claim_Name_Param = config:get(
+        "jwt_auth", "roles_claim_name"
+    ),
+    Roles_Claim_Path_Param = config:get(
+        "jwt_auth", "roles_claim_path"
+    ),
+    Roles =
+        case Roles_Claim_Name_Param of
+            undefined ->
+                % Check for implicitly role_claim "_couchdb.roles" in JWT token
+                case lists:keyfind(<<"_couchdb.roles">>, 1, Claims) of
```

Comment:
I don't understand this. We definitely do not let the token decide where the role param is. The couchdb server's config is the only authority on that.

Reply:
```config:get("jwt_auth", "roles_claim_name", "_couchdb.roles")``` returns the roles_claim_name value from config or, if there isn't one, the default of _couchdb.roles. I see you've done the equivalent logic but in a novel and longer-form way. It would be better to stick with convention and call config:get/3 with our preferred default value.